code/game/objects/items/weapons/storage/blackshield_kits.dm

```diff
@@ -0,0 +1,148 @@
+/obj/item/storage/box/bs_kit
+	name = "\improper Blackshield Kit"
+	desc = "A standard kit."
+	icon_state = "box"
+
+// All-in-one kit
+/obj/item/storage/box/bs_kit/default
+	name = "\improper Default Blackshield Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/boltgun(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
+		new /obj/item/ammo_magazine/speed_loader_rifle_75(src)
```

Comment:
why does it get 40 rounds?

Reply:
Because it is the same amount of the same magazine that Blackshield Troopers start in their lockers